### DISCUSSION TITLE
Panel functions from JSplitter

### DISCUSSION THREAD

**Discussion by dream7180** [TheQwertiest/foo_spider_monkey_panel#216] [Ideas]
It will be great if be able to include panel functions as mod in JSplitter. This is essential to avoid using DUI tab during UI design.
![image](https://github.com/TheQwertiest/foo_spider_monkey_panel/assets/18242221/2be8c8b8-e897-4cb8-967b-40c2251de5b3)

**Comment by TheQwertiest** [comment]
I don't see any reason to implement this in SMP (or JSP for that matter).
SOLID principles not withstanding, it would make more sense (and would be much easier) to implement it as separate panel splitter component.
After that, it would be rather trivial to expose it's interface via COM/ActiveX, which would enable scripting support in all JS components (SMP/JSP/WSH).

**Reply by dream7180** [reply]
I agree with you. A separate component to implement it is preferable. But the problem is that so far no one is available, many themes are still using Panel Stack Splitter (outdated, full of limitations and bugs...).
